### DISCUSSION TITLE
Folding Support?

### DISCUSSION THREAD

**Discussion by DerRockWolf** [artempyanykh/marksman#353] [Q&A] [answered]
Is folding support in scope for marksman or should folding be achieved differently?

**Comment by artempyanykh** [comment, marked as answer]
@DerRockWolf I think it's in scope as per this LSP spec part https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#textDocument_foldingRange

With that said, I'm happy to accept a PR but don't have bandwidth to do the work.
